views/export/index.tt

```diff
@@ -0,0 +1,106 @@
+[%# export/index.tt
+
+Journal export functionality
+
+Authors:
+    Mark Smith <mark@dreamwidth.org>
+
+Copyright (c) 2015 by Dreamwidth Studios, LLC.
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself.  For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%][%- sections.title = title_ml | ml -%]
+
+[%- CALL dw.ml_scope( '/export.bml' ) -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+[% sections.title = '.title' | ml %]
+
+[%- authas_form -%]
+
+<p>[% '.description' | ml %]</p>
+
+<form action='export_do?authas=[% u.username %]' method='post'>
+
+<fieldset>
+
+<div class="row">
+    <div class="large-2 columns"></div>
+    <div class="large-2 columns"><label>[% '.label.what' | ml %]</label></div>
```

Comment:
Specific to the coding of the label: we want the label to be mapped to its relevant form field, e.g.,

    <label for="foo">...</label>
    <input type="..." id="foo">

This helps a lot with accessibility because it makes it programmatically possible to know what the actual label for the form field is for.

It wasn't clear from the style-guide, but (assuming vertical alignment of labels), but you can use the `[% form.x %]`  defined in FormHTML.pm so you don't have to remember all the little details around form elements all the time.